### PR TITLE
style(docs): improve Doc site styles

### PR DESCRIPTION
**Pull request by simeng-li**
## Summary

Docs site styles update

1. Top nav bar style update
image
image

2. Footer update
image

TODO: side bar

@logto-io/eng

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -84,43 +86,28 @@ const config = {
...
           {
-            title: 'Community',
-            items: [
-              {
-                label: 'GitHub',
-                href: 'https://github.com/logto-io/logto',
-              },
-            ],
+            label: 'About Us',
+            href: 'https://logto-io.github.io/website/about',
```
Comment (gao-sun) [inline docusaurus.config.js:103]:
```suggestion
            href: 'https://logto.io/about',
```

File: docusaurus.config.js [hunk 2]
```diff
@@ -32,11 +33,12 @@ const config = {
           breadcrumbs: false,
           sidebarPath: require.resolve('./sidebars.js'),
           // Please change this to your repo.
-          editUrl: 'https://github.com/facebook/docusaurus/tree/main/packages/create-docusaurus/templates/shared/',
+          editUrl:
+            'https://github.com/facebook/docusaurus/tree/main/packages/create-docusaurus/templates/shared/',
```
Comment (gao-sun) [inline docusaurus.config.js:37]: change to ours?

File: docusaurus.config.js [hunk 3]
```diff
@@ -84,43 +86,28 @@ const config = {
...
-              {
-                label: 'Contact Us',
-                href: 'mailto: contact@logto.io'
-              }
-            ],
+            label: 'Contact Us',
+            href: 'mailto: contact@logto.io',
           },
         ],
-        copyright: `Copyright © ${new Date().getFullYear()} Silverhand Inc. Built with Docusaurus.`,
```
Comment (gao-sun) [inline docusaurus.config.js:123]: would like to keep this since we didn't do the major design work of this site

**Review by gao-sun — COMMENTED** [review]
1. let's update PR title
2. will svg logo be better?

File: docusaurus.config.js (hunk 3 above)
Comment (simeng-li) [reply docusaurus.config.js:123]: updated

**Review by wangsijie — APPROVED** [review]

File: docusaurus.config.js [hunk 4]
```diff
@@ -31,12 +32,11 @@ const config = {
           routeBasePath: '/',
           breadcrumbs: false,
           sidebarPath: require.resolve('./sidebars.js'),
-          // Please change this to your repo.
-          editUrl: 'https://github.com/facebook/docusaurus/tree/main/packages/create-docusaurus/templates/shared/',
+          editUrl: 'https://github.com/logto-io/docs/tree/master/docs',
```
[the remark this answers is not in the document]
Comment (simeng-li) [reply docusaurus.config.js]: updated along with svg replacement.

**Pull request merged**